### ISSUE TITLE
500 when accessing French home page

### ISSUE THREAD

**Issue by svbergerem** [diaspora/diaspora-project-site#97]

**Comment by svbergerem** [comment]
```
 Showing /home/steffen/projects/diaspora-project-site/app/views/pages/index.html.erb where line #60 raised:

missing interpolation argument :poduptime in "Trouvez un pod qui vous convient. Cela pourrait être un petit pod ou un situé près de là où vous vivez ou un situé dans un pays où les lois régissant les données privées vous conviennent. C'est vous qui choisissez. Une liste des pods de la communauté est disponible ici %{poduptime}." ({:host_yourself_link=>"<a href=\"https://wiki.diasporafoundation.org/Installation\"><span class=\"translation_missing\" title=\"translation missing: fr.pages.index.host_yourself\">Host Yourself</span></a>"} given)
```

https://webtranslateit.com/en/projects/7017-Website/locales/en..fr/strings/4512446

Seems to be broken for 3 years...

**Comment by Flaburgan** [comment]
@svbergerem that was the point of https://github.com/diaspora/diaspora-project-site/pull/90 translations need to be pushed to webtranslateit.

**Comment by denschub** [comment]
Sorry, I messed up the whole locale thingy. Fixed with fa7ef2edc05f714ec87b4242b1a358500051282c and fdd29e2d055601b79ff2bf5e7514f8c490d7372c.
